shardingsphere-operator/build/tools/Dockerfile

```diff
@@ -24,8 +24,8 @@ ENV ZOOKEEPER_DOWNLOAD_URL https://dlcdn.apache.org/zookeeper/zookeeper-3.7.1/ap
 ENV ZOOKEEPER_DIR /app/zookeeper
 WORKDIR /app
 RUN mkdir -p "/app/start" && chmod -R 777 /app/start
-CMD ["tail -f /dev/null"]
 ENTRYPOINT ["sh","-c"]
```

Comment:
The `entrypoint` should be placed at the last line of the Dockerfile. It is the `entry point` of this docker image. Any commands wants to be executed in this image could be injected by the runtime platform.